lib/net/pool.js

```diff
@@ -4436,7 +4590,9 @@ class PoolOptions {
       assert(Array.isArray(options.only));
       if (options.only.length > 0) {
         this.nodes = options.only;
-        this.maxOutbound = options.only.length;
+        const [standard, brontide] = this.countHostnames(this.nodes);
```

Comment:
I think we should allow for a smaller max outbound passed through the config to take precedence over the number of addrs passed in `only`. E.g. I pass 4 standard peers in `only` but I only want a max of two of them to be connected at a time.

Reply:
I think this functionality is achievable with a combination of `--nodes` and `--max-outbound`. If not, maybe we can focus on this in the future.